src/components/SongRecommendation/SongRecommendation.js

```diff
@@ -0,0 +1,96 @@
+import { useState, useEffect } from "react";
+import { Col, ListGroupItem, Row } from "react-bootstrap";
+import ListGroup from 'react-bootstrap/ListGroup';
+import "./SongRecommendation.css";
+
+const SongRecommendation = (props) => {
+
+    const PLAYLISTS_ENDPOINT = "https://api.spotify.com/v1/playlists/";
+    const ACCESS_TOKEN_ENDPOINT = 'https://accounts.spotify.com/api/token';
+    var accessToken = '';
+    var playlistId = '';
+    const [tracksData, setTracksData] = useState(null);
+    // eslint-disable-next-line no-lone-blocks
```

Comment:
This comment probably needs to be removed.

Reply:
Okay. Will look into it 👍